ngcc.config.js

```diff
@@ -21,5 +21,14 @@ module.exports = {
         },
       },
     },
+    'ng2-pdfjs-viewer': {
```

Comment:
Can you, please, add a comment explaining what this config does/why it is needed?

Reply:
I removed it altogether since there are additional things wrong with the library and I have provided them with fixes.